src/Factory/Api/Mastodon/FollowRequest.php

```diff
@@ -45,17 +44,18 @@ public function __construct(LoggerInterface $logger, BaseURL $baseURL)
 
 	/**
 	 * @param Introduction $introduction
-	 * @return \Friendica\Object\Api\Mastodon\FollowRequest
+	 * @return \Friendica\Object\Api\Mastodon\Account
 	 * @throws ImagickException|HTTPException\InternalServerErrorException
 	 */
-	public function createFromIntroduction(Introduction $introduction): \Friendica\Object\Api\Mastodon\FollowRequest
+	public function createFromIntroduction(Introduction $introduction): \Friendica\Object\Api\Mastodon\Account
```

Comment:
Please move this method to `src/Factory/Api/Mastodon/Account.php` since it now returns a `Object\Api\Mastodon\Account` object.

It may remove the need for this whole class.